src/Tool/ORM/Walker/SqlWalkerCompatForOrm2.php

```diff
@@ -45,15 +47,11 @@ public function getExecutor($statement)
     }
 
     /**
-     * Walks down a SelectStatement AST node, thereby generating the appropriate SQL.
-     *
-     * @param SelectStatement $selectStatement
-     *
-     * @return string
+     * @param DeleteStatement|UpdateStatement|SelectStatement $AST
      */
-    public function walkSelectStatement($selectStatement)
```

Comment:
Sorry for the lack of context, I'm just returning to my desk.

Isn't this removal a potential breaking change for existing implementations?

Reply:
If someone's using this `@internal` tagged trait directly, yes.  These compat traits weren't meant to be public APIs though, they're here to help with the ORM 2 vs 3 compat layer.